README.rst

```diff
@@ -39,12 +39,18 @@ The Public Utility Data Liberation Project (PUDL)
 .. |kaggle| image:: https://img.shields.io/badge/Kaggle-20BEFF?style=flat&logo=Kaggle&logoColor=white
    :target: https://www.kaggle.com/datasets/catalystcooperative/pudl-project
    :alt: The PUDL Dataset on Kaggle
+.. |youtube| image:: https://img.shields.io/badge/YouTube-%23FF0000.svg?logo=YouTube&logoColor=white
+   :target: https://youtube.com/@CatalystCooperative
+   :alt: Catalyst Cooperative on YouTube
+.. |twitter| image:: https://img.shields.io/badge/X-%23000000.svg?logo=X&logoColor=white
+   :target: https://x.com/CatalystCoop
+   :alt: Catalyst Cooperative on Twitter
```

Comment:
Re-added Twitter since I am still posting there. Not sure if we want to continue with that, but there are still people who haven't left.

Reply:
T_T